app/controllers/application_controller.rb

```diff
@@ -1,6 +1,12 @@
 class ApplicationController < ActionController::Base
-  before_action :authenticate_user!
+  include Pundit
+
+  after_action :verify_authorized, except: :index
```

Comment:
- This method is a safeguard for everyone working on an app, to make sure that we do not forget to set up policies. We should probably call `verify_authorized` on index also.
- You might want to add Devise as an exception though `, unless: :devise_controller?`

Reply:
fixed